server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/CompactableImpl.java

```diff
@@ -679,6 +688,8 @@ public void compact(CompactionServiceId service, CompactionJob job, RateLimiter
         selectedCompactionCompleted(job, jobFiles, metaFile);
       else
         selectFiles();
+
+      tablet.updateTimer(MAJOR, queuedTime, startTime, stats.getEntriesRead(), failed);
```

Comment:
Could shorten the code a bit by removing failed boolean.   Please ignore this suggestion if it does not suit you. Just something I thought of when I looked at the code.

```suggestion
      tablet.updateTimer(MAJOR, queuedTime, startTime, stats.getEntriesRead(), metaFile == null);
```

Reply:
I like it. As long as it doesn't go over the char line limit... *drum roll*

Reply:
98 chars!